app/src/app/pages/artwork/artwork.component.html

```diff
@@ -1,5 +1,7 @@
 <div class="container entity artwork">
   <div class="col pb-5 border-bottom">
+    <app-wrong-id *ngIf="idDoesNotExist" [id]="artworkId"></app-wrong-id>
```

Comment:
Make sure to hide all other content when idDoesNotExist is true (applies to all entity pages)